File: src/main/java/org/telran/lecture_9_linked/hw/Node.js
class Node {
    constructor(data) {
        this._next = null;
        this._prev = null;
        this._data = data;
    }

    get data() {
        return this._data;
    }

    set data(value) {
        this._data = value;
    }

    get next() {
        return this._next;
    }

    set next(value) {
        this._next = value;
    }

    get prev() {
        return this._prev;
    }

    set prev(value) {
        this._prev = value;
    }
}

File: src/main/java/org/telran/lecture_9_linked/hw/Node.java
package org.telran.lecture_9_linked.hw;

public class Node {
    private int data;
    private Node next;
    private Node prev;

    public Node(int data, Node next, Node prev) {
        this.data = data;
        this.next = next;
        this.prev = prev;
    }

    public int getData() {
        return data;
    }

    public void setData(int data) {
        this.data = data;
    }

    public Node getNext() {
        return next;
    }

    public void setNext(Node next) {
        this.next = next;
    }

    public Node getPrev() {
        return prev;
    }

    public void setPrev(Node prev) {
        this.prev = prev;
    }
}
